README.md

```diff
@@ -148,6 +148,22 @@ To run linter tests, run the following command:
 - Twitter: [@rodrigopenayo98](https://twitter.com/rodrigopenayo98)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/rodrigo-penayo-391226158/)
 
+

```

Comment:
- [ ] Great work so far 👍 but i guess you didn't understand the requirements well 🤔 It is a pair-programming project means every member of this team should commit code from his/her side into the codebase but i see that only one member pushed code but other two members only fix the linters and readme file but didn't push any code from their side 😨 Kindly all members push some code into this pull request to fulfill the pair-programming requirement 👍

Reply:
hello @HAMMAS-SALEEM ! how are you? Look, we all collaborated... there wasn't much to do, the concept of pair-programming does not necessarily indicate that everyone must add code, but rather that they collaborate on the same project.
Anyway, the girls will add code in the following group projects! we will rotate. We work this way on many occasions and it is the first time we receive that type of correction.